test/parse/test_entity_declaration.rb

```diff
@@ -23,6 +23,239 @@ def parse(internal_subset)
     end
 
     public
+
+    class TestGeneralEntityDeclaration < self
+      # https://www.w3.org/TR/2006/REC-xml11-20060816/#NT-GEDecl
+      class TestEntityDefinition < self
+        # https://www.w3.org/TR/2006/REC-xml11-20060816/#NT-EntityDef
+        class TestEntityValue < self
+          def test_no_quote
+            # https://www.w3.org/TR/2006/REC-xml11-20060816/#NT-EntityValue
+            exception = assert_raise(REXML::ParseException) do
+              REXML::Document.new('<!DOCTYPE root [<!ENTITY valid-name invalid-entity-value > ]>')
+            end
+            assert_equal(<<-DETAIL.chomp, exception.to_s)
+Malformed entity declaration
+Line: 1
+Position: 61
+Last 80 unconsumed characters:
+ valid-name invalid-entity-value > ]>
+            DETAIL
+          end
+
+          def test_invalid_entity_value
```

Comment:
Could you use why this is an invalid value for test name instead of `test_invalid_entity_value` like you did for `test_no_quote`?

Reply:
Changed the method name at https://github.com/ruby/rexml/pull/183/commits/98e0d8fcc7cf6ed30510ff49b34f04bc02431725